scripts/package

```diff
@@ -73,6 +73,9 @@ if [ ${ARCH} == amd64 ]; then
     # Move this out of ARCH check for local dev on non-amd64 hardware.
     TAG=$TAG REPO=${REPO} go run ../pkg/image/export/main.go $SYSTEM_CHART_REPO_DIR $CHART_REPO_DIR/assets $IMAGE $AGENT_IMAGE $RUNTIME_IMAGE
 
+    # CVEs CSV
+    curl -sL https://raw.githubusercontent.com/rancher/image-scanning/main/docs/_data/cves-v2.6.csv -o rancher-v2.6-cves.csv
```

Comment:
nit: might want to use args like how we do for $CHART_REPO_*.

Reply:
We should rename that file to `${TAG}_CVE_SCAN_RESULTS` or similar.